lib/broccoli/translation-reducer/linter/index.js

```diff
@@ -44,25 +44,31 @@ module.exports = class Linter {
 
     // create flattened list of all unique translation keys
     const allTranslationKeys = new Set(Array.prototype.concat(...localeKeys.map(([, keys]) => keys)));
-    const { requiresTranslation, fallbackLocale } = this.options;
+    const { requiresTranslation } = this.options;
 
     allTranslationKeys.forEach((key) => {
-      const notInLocales = findMissingTranslations(key, localeKeys, requiresTranslation);
+      const translationMissingFromLocales = findLocalesMissingTranslation(key, localeKeys);
 
-      if (notInLocales.length) {
-        result.missingTranslations.push([key, notInLocales]);
-      }
-
-      let missingICUArgs = findMissingICUArguments(key, allIcuArguments, locales, icuArguments);
+      if (translationMissingFromLocales.length) {
+        const alertableMissingTranslations = translationMissingFromLocales.filter((locale) =>
+          requiresTranslation(key, locale)
+        );
 
-      if (fallbackLocale) {
```

Comment:
This logic isn't right.  If there is a `fallbackLocale` it shouldn't have a side effect of disabling linting other locales for mismatch ICU arguments.